### ISSUE TITLE
Applying rotation to 2D sprite makes it dissapear

### ISSUE THREAD

**Issue by ArtBIT** [SebLague/Path-Creator#109]
If I try to follow the examples and use the path rotation at distance for 2D sprites, the sprite disappears.
```
transform.rotation = pathCreator.path.GetRotationAtDistance(distanceTravelled);
```
This is what I had to do to achieve the desired behavior for 2D objects:

```
transform.rotation = Quaternion.FromToRotation(Vector3.right, pathCreator.path.GetNormalAtDistance(distanceTravelled));
```

**Comment by brenden-t-r** [comment]
Agreed, MathUtility.TransformDirection doesn't seem to return the correct result in the 2D scenarios, despite appearing to have code for this at a quick glance. Unless perhaps my setup is wrong in some way. The solutions above work or more generally you can modify VertexPath.GetRotation to the Quaternion `rotation` value before returning:

```
if (this.space == PathSpace.xy) {
    rotation.x = 0f;
    rotation.y = 0f;
} else if (this.space == PathSpace.xz) {
    rotation.x = 0f;
    rotation.z = 0f;
}
```

**Comment by ShaneYu** [comment]
Excellent, thanks @ArtBIT - Although in my case, I had to use `Vector3.up` opposed to `Vector3.right`. ❤️
